### PR TITLE
doc/releases/schedule: add mimic column

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

**Comment by dillaman** [comment]
jenkins render docs

**Comment by liewegas** [comment]
jenkins render docs

File: doc/releases/mimic.rst [hunk 1]
```diff
@@ -162,30 +162,33 @@ Upgrading from pre-Luminous releases (like Jewel)
...
     Luminous monitor is running along with a Mimic OSD. Please use the
     recommended upgrade order of monitors before OSDs to avoid this issue.
 
   - The sample ``crush-location-hook`` script has been removed. Its output is
     equivalent to the built-in default behavior, so it has been replaced with an
     example in the CRUSH documentation.
 
-  - The -f option of the rados tool now means "--format" instead of "--force",
-    for consistency with the ceph tool.
+  - The `-f` option of the rados tool now means `--format` instead
```
Comment (theanalyst) [inline doc/releases/mimic.rst]: double backticks for --format and force maybe

Comment (liewegas) [reply doc/releases/mimic.rst]: updated.  really need to sort out what the rules are for this, though.. i was using "it is a shell command line thing" for double ticks and single ticks for everything else, but there is probably a real rule we should be using?

Comment (LenzGr) [reply doc/releases/mimic.rst]: Note that single backticks in reStructuredText have a different meaning than in Markdown - in the latter, they mean "print this on monospace", which is usually used for highlighting shell commands or commandline options in text blocks.

In [reST markup](http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup), you would use double backticks to achieve the same formatting - single backticks have a special meaning there.

**Pull request merged**